### PR TITLE
mostly copy edits

### PR DESCRIPTION
**Pull request by nayana487**

### CODE REVIEW

File: attendance.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
 
 **There are no exceptions, even if you forgot and the instructors saw you were here.**
 
 #### Checkpoints
 
-So as not to disrupt their peers, tardy students should wait until the 10 am morning lesson to enter the classrooms. The ONLY exception to this policy is when a checkpoint is being administered. If a student is up to 10 minutes late, they're asked to take the quiz in their assigned classroom. Otherwise, the checkpoint should be completed during lunch and the PR should be submitted no later than 2:30 pm. Tardies will still apply, but credit will be received for taking the checkpoint.
+As not to disrupt their peers, tardy students should wait until the 10 am morning lesson to enter the classrooms. The ONLY exception to this policy is when a checkpoint is being administered. If a student is up to 10 minutes late, they're asked to take the quiz in their assigned classroom. Otherwise, the checkpoint should be completed during lunch and the PR should be submitted no later than 2:30 pm. Tardies will still apply, but credit will be received for taking the checkpoint.
```
Comment (amaseda) [inline attendance.md]: "they're asked" --> "they will be asked"

Not sure, but sounds kinda weird as is, right? Could be technically correct.

File: checkpoints.md [hunk 2]
```diff
@@ -1,8 +1,6 @@
 # Checkpoints
 
-Here's a list of need to know concepts covered in each of the various weeks. Each week, we'll compile a list of everything we absolutely need to know moving forward as web developers. That is not to say that everything else we cover in class isn't important, but the things listed here are the absolute minimum requirements in order to be successful. If you find yourself looking over past weeks and are forgetting or misremembering a concept, you are highly encouraged to make sure to understand them throughout the entirety of the course(and as a developer).
-
-## Week 1
+Here's a list of need to know concepts covered in each of the various weeks. We've compiled a list of everything we absolutely need to know moving forward as web developers. That is not to say that everything else we cover in class isn't important, but the things listed here are the absolute minimum requirements in order to be successful. If you find yourself looking over past weeks and are forgetting or misremembering a concept, you are highly encouraged to make sure to understand them throughout the entirety of the course (and as a developer).
```
Comment (amaseda) [inline checkpoints.md]: "need to know" --> "need-to-know"

File: graduation-requirements.md [hunk 3]
```diff
@@ -5,7 +5,7 @@
 - Complete and present each of WDI's four projects
 - Be absent no more than 2 days over the whole course
 - Be tardy no more than 5 days over the whole course
-- Complete the [Job Seeker Toolkit before Outcomes Bootcamp](https://docs.google.com/document/d/1dTtTYLM6Ev6XUvmWm4iuxhC-wTMuTllqXR2CNoDJ_Ik/edit#heading=h.vrvrq0gile16)
+- Complete 100% of [Outcomes homework](https://github.com/ga-dc-outcomes) throughout programming
```
Comment (amaseda) [inline graduation-requirements.md]: Don't think you need to specify "throughout programming"

File: homework-policy.md [hunk 4]
```diff
@@ -2,76 +2,75 @@
...
 
 ## What to include with your submission
 
 On **every submission** -- that is, on every pull request (or, sometimes, issue) -- please include...
 - **Comfort score**, out of 5
 - **Completeness score**, out of 5
 
+### In order to get feedback on your assignment...
+
+...when submitting your pull request or issue, please...
```
Comment (amaseda) [inline homework-policy.md:43]: "...when" --> "When"

File: attendance.md (hunk 1 above)
Comment (nayana487) [reply attendance.md]: "they should"

File: checkpoints.md (hunk 2 above)
Comment (nayana487) [reply checkpoints.md]: 👍

File: graduation-requirements.md (hunk 3 above)
Comment (nayana487) [reply graduation-requirements.md]: 👍

File: homework-policy.md (hunk 4 above)
Comment (nayana487) [reply homework-policy.md:43]: no

File: asking-for-help.md [hunk 5]
```diff
@@ -9,16 +9,14 @@ We ask that you take the following steps when dealing with technical questions..
 1. Try something else
 2. Google your exact problem or error
 3. Reference the appropriate lesson plan
-4. Ask another classmate
-5. Ask an instructor
+4. Ask your classmates
+5. Ask an instructor during scheduled office hours and one-on-ones
```
Comment (superbuggy) [inline asking-for-help.md:14]: and class too!

File: before-wdi.md [hunk 6]
```diff
@@ -1,11 +0,0 @@
...
-
-**Please complete the following steps as soon as possible.** We can't make classroom/squad assignments until you do!
-
-1. **Click [this link](https://garnet.wdidc.org/github/authorize?invite_code=a5d4bd0222d19c66a146d4b225f8e3ea).** This will authorize you for Garnet, an application that both instructors and students will use to track your progress throughout WDI.  
-
-2. **Update your GitHub profile with your full name and profile picture.** The latter is very important - we need to know what you look like! The clearer the picture, the better.
-
-Once you're done, we encourage you to take a look through [this readme](https://github.com/ga-dc/WDI15/blob/master/readme.md) if you would like to learn more about what you can expect out of WDI.
-
-See you all at orientation!
```
Comment (superbuggy) [inline before-wdi.md:11]: Are we adding this back in later? If so, let's have placeholder text. Why is this unnecessary here?

File: homework-policy.md [hunk 7]
```diff
@@ -2,76 +2,75 @@
...
 js1989: ~/wdi/homeworkaroo (master) $ git checkout -b johnsmith_solution
-js1989: ~/wdi/homeworkaroo (master) $ touch did_my_homework.txt
-js1989: ~/wdi/homeworkaroo (master) $ git add did_my_homework.txt
-js1989: ~/wdi/homeworkaroo (master) $ git commit -m "Added did_hw file. All done"
-js1989: ~/wdi/homeworkaroo (master) $ git push origin master
+js1989: ~/wdi/homeworkaroo (johnsmith_solution) $ touch did_my_homework.txt
+js1989: ~/wdi/homeworkaroo (johnsmith_solution *) $ git add did_my_homework.txt
+js1989: ~/wdi/homeworkaroo (johnsmith_solution +) $ git commit -m "Added did_hw file. All done"
+js1989: ~/wdi/homeworkaroo (johnsmith_solution) $ git push origin johnsmith_solution
 # Click green 'New pull request' button on Github
```
Comment (superbuggy) [inline homework-policy.md:73]: that's what's up 👌

File: homework-policy.md [hunk 8]
```diff
@@ -2,76 +2,75 @@
...
+3. Create a feature branch in the format of `yourname_solution`.
+4. Complete your work inside it.
+5. Then, **push** your completed work to your forked repository.
+6. Finally, make a **pull request** to the upstream repository (in `ga-wdi-exercises`).
 
 For example, the sequence of commands you might follow to complete the above process is:
 
 ```
-# Click grey 'Fork' button on Github
+# Click gray 'Fork' button on Github
```
Comment (superbuggy) [inline homework-policy.md:65]: :woman_firefighter:  no chill

File: homework-policy.md [hunk 9]
```diff
@@ -2,76 +2,75 @@
...
 ## The Submission Process
 
 All homework assignments will have their own Github repository, under the `ga-wdi-exercises` account.
 
 1. Please **fork** new assignments.
 2. Then, **clone** your fork to your computer.
-3. Add and commit your work.
-4. Then, **push** your completed work to your forked repository.
-5. Finally, make a **pull request** to the upstream repository (in `ga-wdi-exercises`).
+3. Create a feature branch in the format of `yourname_solution`.
```
Comment (superbuggy) [inline homework-policy.md:57]: 👍

File: readme.md [hunk 10]
```diff
@@ -16,7 +16,7 @@
 - [All WDI Exercises](http://repotagger.github.io?name=ga-wdi-exercises)
 - [All WDI Lessons](http://repotagger.github.io?name=ga-wdi-lessons)
 - [Installfest](https://github.com/ga-dc/installfest)
-- [Outcomes Roadmap](https://github.com/ga-dc/outcomes/tree/master/roadmap)
+- [Outcomes](https://github.com/ga-dc-outcomes)
```
Comment (superbuggy) [inline readme.md:20]: ✔️

**Pull request closed without merging**